functions.php

```diff
@@ -180,6 +180,7 @@ function siteorigin_corp_scripts() {
 
 	// Flexslider.
 	wp_register_script( 'jquery-flexslider', get_template_directory_uri() . '/js/jquery.flexslider' . SITEORIGIN_THEME_JS_PREFIX . '.js', array( 'jquery' ), '2.6.3', true );
+	wp_register_style( 'siteorigin-corp-flexslider', get_template_directory_uri() . '/css/flexslider' . SITEORIGIN_THEME_CSS_PREFIX . '.css', array(), '2.6.3' );
```

Comment:
Should we not be using a standard handle instead of `siteorigin-corp-flexslider` if we're using the standard Flexslider stylesheet?

Reply:
This line was partially copied from Unwind so I kept the same naming convention for the slug. I'll change this in the morning.

Reply:
Super, thanks.